### PR TITLE
PMM-5926 Add support for pmm-agent version.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/pmm-agent?pullRequest=161) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/pmm-agent?pullRequest=161) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/pmm-agent?pullRequest=161) it.</sub>

File: agentlocal/agent_local.go [hunk 1]
```diff
@@ -135,10 +136,11 @@ func (s *Server) Status(ctx context.Context, req *agentlocalpb.StatusRequest) (*
...
 		serverInfo = &agentlocalpb.ServerInfo{
-			Url:         u.String(),
-			InsecureTls: s.cfg.Server.InsecureTLS,
-			Connected:   connected,
-			Version:     md.ServerVersion,
+			Url:          u.String(),
+			InsecureTls:  s.cfg.Server.InsecureTLS,
+			Connected:    connected,
+			Version:      md.ServerVersion,
+			AgentVersion: version.Version,
```
Comment (BupycHuk) [inline agentlocal/agent_local.go:143]: AgentVersion should not be in ServerInfo, because ServerInfo is about PMM-Server

Comment (JiriCtvrtka) [reply agentlocal/agent_local.go]: @BupycHuk I moved it into StatusResponse, please check if it is OK. Thank you.

File: Gopkg.toml [hunk 2]
```diff
@@ -26,7 +26,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-5926-agent-version"
```
Comment (AlekSi) [inline Gopkg.toml:29]: It was merged

**Pull request merged**